### ISSUE TITLE
Duplicate floats in same selector aren't always removed.

### ISSUE THREAD

**Issue by hostep** [jakubpawlowicz/clean-css#540]
Using cleancss 3.2.2

This is correct:
```
➜ echo 'p {float: right; float: right;}' | ./cleancss
p{float:right}
```

This is not what I expected:
```
➜ echo 'p {float: left; float: right;}' | ./cleancss
p{float:left;float:right}
```
It should output:
```
p{float:right}
```

This is also not what I expected:
```
➜ echo 'p {float: right; float: right !important;}' | ./cleancss
p{float:right;float:right!important}
```
It should output:
```
p{float:right!important}
```

The last case worked in 3.1.9 as expected (I didn't test it explicitly, but I can see it from the diffs of a very large file)

**Comment by jakubpawlowicz** [comment]
It's still correct though. Let me see if we can restore the old functionality, otherwise it will have to wait for #290.

**Comment by jakubpawlowicz** [comment]
It waits for #290 to be implemented in 3.5. Closing as duplicate.
